test/org/openlcb/implementations/BitProducerConsumerTest.java

```diff
@@ -605,8 +634,14 @@ private void createWithDefaults() {
         expectNoFrames();
     }
 
-    @Override
+    @Before
     public void setUp() {
+        super.setUp();
         aliasMap.insert(0x444, new NodeID(new byte[]{1,2,3,1,2,3}));
     }
-}
\ No newline at end of file
+
+    @After
```

Comment:
QQ. Is this necessary or could we rely on the base class `@After` method to execute the teardown?